### PR TITLE
HIMAL-370 Move Activty API to external API

### PR DESCRIPTION
**Pull request by mark-a-johnson**
HIMAL-366 Add separate counters for Instances vs. Service Instances

### CODE REVIEW

**Comment by servicemesh-autobuild** [comment]
:white_check_mark: Build was successful :tada:


http://172.22.12.115:8080/job/csc-agility-platform-sdk-prb/284/

File: agility.api/src/main/java/com/servicemesh/agility/api/activity/service/IActivityManager.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (C) 2016 Computer Science Corporation
+ * All rights reserved.
+ *
+ */
```
Comment (sganesanmuth) [inline agility.api/src/main/java/com/servicemesh/agility/api/activity/service/IActivityManager.java]: I guess it's outdated Copyright message

**Comment by sganesanmuth** [comment]
Looks like I don't have write access to this repo.

File: agility.api/src/main/java/com/servicemesh/agility/api/activity/service/IActivityManager.java (hunk 1 above)
Comment (mark-a-johnson) [reply agility.api/src/main/java/com/servicemesh/agility/api/activity/service/IActivityManager.java]: Yes, this was copied from the core code as is.  Sometime we need to go over these and update them.

**Pull request merged**